### PR TITLE
ConfigTrait unit tests

### PR DESCRIPTION
**Pull request by oldDriver**
# Description
Added unit tests which cover 100% of the ConfigTrait and fixed one fatal error

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [+] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [+] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [+] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: tests/unit/testsuite/Bolt/Boltpay/Helper/ConfigTraitTest.php [hunk 1]
```diff
@@ -1,66 +1,946 @@
...
+                    'expect' => array('1', '2', '5', '6'),
+                    'extra_options' => '{"datadogKeySeverity": "1,2,              5                , 6"}'
+                )
+            ),
+        );
+    }
+
+    public function tearDown()
+    {
+        //Mage::reset();
```
Comment (daisy1754) [inline tests/unit/testsuite/Bolt/Boltpay/Helper/ConfigTraitTest.php]: remove?

Comment (oldDriver) [reply tests/unit/testsuite/Bolt/Boltpay/Helper/ConfigTraitTest.php]: @daisy1754 Yes!!!
This line crushed unit tests chain with
PHP Fatal error:  Cannot redeclare benchmark() (previously declared in /home/aymelyanov/www/magento-one/app/code/community/Bolt/Boltpay/Model/Observer.php:48)

File: tests/unit/testsuite/Bolt/Boltpay/OrderHelper.php [hunk 2]
```diff
@@ -145,7 +145,11 @@ public static function deleteDummyOrderByIncrementId($incrementId)
      */
     public static function deleteDummyOrder(Mage_Sales_Model_Order $order)
     {
-        self::deleteDummyOrderByIncrementId($order->getIncrementId());
-
+        if (!empty($order)) {
+            $id = $order->getIncrementId();
+            if (!empty($id)) {
```
Comment (mccottry) [inline tests/unit/testsuite/Bolt/Boltpay/OrderHelper.php]: When is it in our test that we have an order without an `incrementId`?

Here, we appear to be simply hiding errors in our test and should instead fix the erroneous testing conditions instead of allowing them to persist, and also, create a more robust and comprehensive `deleteDummyOrder` function.

Since we know that `$order` will be an instance of `Mage_Sales_Model_Order`  we can call `Mage_Sales_Model_Order::delete()`.  This is more robust and comprehensive as it doesn't require the order to have an increment id, and at the same time cleans up the orders that do not have this instead of leaving them unhandled as in this code. 

Also, since

File: tests/unit/testsuite/Bolt/Boltpay/OrderHelper.php [hunk 3]
```diff
@@ -145,7 +145,11 @@ public static function deleteDummyOrderByIncrementId($incrementId)
...
+                self::deleteDummyOrderByIncrementId($id);
+            }
+        }
```
Comment (mccottry) [inline tests/unit/testsuite/Bolt/Boltpay/OrderHelper.php]: This appears to be unrelated to the subject of the PR and should be in it's own PR.
